templates/log-in-form.html

```diff
@@ -44,6 +44,9 @@
             <input name="log-in.password" class="form-control"
                    type="password" autocomplete="current-password"
                    placeholder="{{ _('Password') }}" required />
+            <input name="log-in.totp" class="form-control"
```

Comment:
I'd prefer that myself, but that would require a bit more rework in the auth-logic than I was comfortable with when I first set out. (But since I had to do the dumb javascript-hack in order to feed the confirm-link with an otp, I guess I might as well bite the bullet and make it proper.)
That said, I'm trying to understand the `user_secrets`-table since it seem to both contain hashed+salted passwords and sessions. I guess keeping the authentication state there would make sense, but then keeping the totp-info in the two separate columns in `participants` should be reconsidered as well, given that more 2FA-methods might be a thing.